### ISSUE TITLE
mjml should auto-complete tags

### ISSUE THREAD

**Issue by connor4312** [mjmlio/vscode-mjml#14]
Before you open an issue, please check if a similar issue already exists or has been closed before.

## Expected behavior

I can type `<mj-` and see a list of completed mj- tags

## Actual behavior

No completions are provided

## Steps to reproduce the problem

  1. Type `<mj-`
  1. Trigger completions

## Code sample

```
<mj-
```

## Screenshot / GIF

![](https://memes.peet.io/img/21-03-7314c63f-d524-4677-92a6-0a55e11a051e.gif)

## Specifications

  - VS Code version: 1.55.0-insiders
  - MJML extension version: 1.0.4
  - Operating system and version: Win 10

## Other information

List any other information that is relevant to your issue.

**Comment by iRyusa** [comment]
You don't really need to type `<` anymore just type `mjcolumn`

**Comment by connor4312** [comment]
I realize that, but this is different from how HTML and its derivatives work in vscode. I would expect mjml to work the same way.

**Comment by iRyusa** [comment]
Yeah this was designed like that feel free to open a pr to have both alternative then

**Comment by dazgreer** [comment]
Added snippets for all `<mj-` tags so should now autocomplete  (in v2.3.0)
